### PR TITLE
Display completed badge when work history is completed

### PR DESCRIPTION
**Pull request by tvararu**
### Context

Title says it all really.

### Changes proposed in this pull request

Add a new field to the model, update it, test it.

### Guidance to review

Screenshots:

Screenshot 2019-10-31 at 18 57 07
Screenshot 2019-10-31 at 18 57 13

### Link to Trello card

[192 - Adding work history](https://trello.com/c/G8slDphi/192-adding-work-history)

### CODE REVIEW

File: spec/system/candidate_interface/candidate_entering_work_history_spec.rb [hunk 1]
```diff
@@ -132,4 +137,20 @@ def and_i_change_the_job_title
...
+  def when_i_mark_this_section_as_completed
+    check t('application_form.work_history.review.i_have_completed')
+  end
+
+  def and_i_click_on_continue
+    click_button t('application_form.work_history.review.button')
+  end
+
+  def then_i_should_see_the_form
+    expect(page).to have_content(t('page_titles.personal_details'))
```
Comment (chubberlisk) [inline spec/system/candidate_interface/candidate_entering_work_history_spec.rb]: Don't think this is what we should expect.

File: config/locales/application_form.yml [hunk 2]
```diff
@@ -80,6 +80,9 @@ en:
       delete_entry: Delete entry
       add_job: Add job
       sure_delete_entry: Yes I’m sure - delete this entry
+      review:
+        i_have_completed: I have completed this section
```
Comment (chubberlisk) [inline config/locales/application_form.yml]: Maybe `completed_checkbox`?

File: app/views/candidate_interface/work_history/review/show.html.erb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
 
 <h1 class="govuk-heading-xl">
-  <%= page_title('work_history') %>
+  <%= t('page_titles.work_history') %>
```
[the remark this answers is not in the document]
Comment (tvararu) [reply app/views/candidate_interface/work_history/review/show.html.erb]: Close enough! :)

File: spec/system/candidate_interface/candidate_entering_work_history_spec.rb (hunk 1 above)
Comment (tvararu) [reply spec/system/candidate_interface/candidate_entering_work_history_spec.rb]: Oh yeah!

**Pull request merged**